### PR TITLE
New formula: protobuf-swift installation using homebrew

### PR DESCRIPTION
**Pull request by alexeyxo**
An implementation of Protocol Buffers in Swift.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Is this not better suited in Carthage/CocoaPods instead?

File: Library/Formula/protobuf-swift.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on "protobuf"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
```
Comment (mikemcquaid) [inline Library/Formula/protobuf-swift.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by alexeyxo** [comment]
Cocoapods/Carthage used for installing library. But this compiler.
For example: protobuf-c formula.

File: Library/Formula/protobuf-swift.rb (hunk 1 above)
Comment (alexeyxo) [inline Library/Formula/protobuf-swift.rb]: Done. Thanks.

**Comment by mikemcquaid** [comment]
A few failing audit rules: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/25004/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_protobuf_swift___strict/

**Comment by alexeyxo** [comment]
Hi! What do you think of merge this formula? Thanks.

**Comment by jacknagel** [comment]
Do you anticipate a lot of people will use Homebrew to install this? Isn't it more likely they will use something like carthage or cocoapods?

**Comment by alexeyxo** [comment]
@jacknagel
This project consists of two parts - runtime-library and compiler.  "Carthage" and "cocoapods" used for installation runtime-library. Compiler can be installed from homebrew.

**Comment by DomT4** [comment]
If still interested in this can you add a `desc` above the `homepage` and squash your commits, Thanks!

**Comment by alexeyxo** [comment]
@DomT4 Done. Thanks.

**Comment by DomT4** [comment]
Jack/Mike, feelings on this?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `protobuf-swift 1.6.2 (new formula)`.

**Comment by alexeyxo** [comment]
Thanks, guys!

**Pull request closed without merging**